src/views/EditEvent.tsx

```diff
@@ -11,10 +11,10 @@ import ReturnHome from "../components/ReturnHome";
 const EditEvent = () => {
   // get id from URL
   const path = useLocation().pathname;
-  const id = path.split("/")[2];
+  const id = path.split("/")[2]; // TODO - split to method, add unit test - any better way to do this maybe like useParams?
```

Comment:
Seeing hardcoded numbers makes me nervous that this is going to be brittle.  If that's the case, refactor it's it's own method that produces the ID and add tests for that method describing what input you expect and what output you'll get.

Though I think you may be able to find a way to get this via `useParam`